test/physics/em/ImportedProcesses.test.cc

```diff
@@ -196,6 +197,40 @@ TEST_F(ImportedProcessesTest, photoelectric)
     }
 }
 
+TEST_F(ImportedProcessesTest, bremsstrahlung)
+{
+    // Create photoelectric process (requires Geant4 environment variables)
+    std::shared_ptr<BremsstrahlungProcess> process;
+    try
+    {
+        process = std::make_shared<BremsstrahlungProcess>(
+            particles_, materials_, processes_);
+    }
+    catch (const RuntimeError& e)
+    {
+        SKIP("Failed to create process: " << e.what());
+    }
+
+    // Test model
+    auto models = process->build_models(ModelIdGenerator{});
+    ASSERT_EQ(1, models.size());
+    ASSERT_TRUE(models.front());
+    EXPECT_EQ("Seltzer-Berger bremsstrahlung", models.front()->label());
+    auto all_applic = models.front()->applicability();
+    ASSERT_EQ(1, all_applic.size());
```

Comment:
I think the size should be 2 (both electron and positron).

Reply:
I have a feeling this test doesn't pass yet ;)

Reply:
Right. I tried running this test but apparently didn't configure my celeritas build to use ROOT so it doesn't run....